### PR TITLE
Vam 26 gmv param

### PR DESCRIPTION
**Pull request by sidjtd**
Context.name (line 84) needs to be re-factored to utilize User Service but other than that, hopefully all is good to go!!!!

(removed a lot of returns in front of callbacks to make them one liners... please lmk if that was not a good idea, reverting it back to all having 'return callbacks()' is too easy!

### CODE REVIEW

File: lib/broker/models/gmv-parameter.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            'GMV ALLOW USER TEMPLATES',
+            'GMV DEFAULT VALUES',
+            'GMV TEMPLATE',
+            'GMV TEMPLATE DEFAULT',
+            'GMV USER DEFAULTS',
+        ],
+        files: ['8989.3', '8989.51', '8989.518'],
+        domain: 'CPRS',
+        labels: ['META', 'VITAL', 'CHANGE'],
+        services: [],
```
Comment (mfuroyama) [inline lib/broker/models/gmv-parameter.js]: Mr. Pedantic reluctantly mumbles: let's add 'cprsMetaService' here... 😄

File: lib/services/cprsMetaService/gmvParameter.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+/**
+ * @summary Sets and retrieves parameter values used by the graphical user interface.
+ * @param {String} routineArg Routine tag line to call
+ * @param {String} entity The entity value to use
+ * @param {String} gmvParamString Parameter string used to call parameter service
+ * @param {String} instance The instance to use.
+ * @param {String} value The value assigned to a parameter, stored in FILE 8989.5
+ * @param  {Object}   context  Session context information associated with the service call.
+ * @param  {Function} callback Function called on process completion, with standard (err, result) signature.
+ * @return {[type]}            [description]
```
Comment (mfuroyama) [inline lib/services/cprsMetaService/gmvParameter.js]: I'm guessing but I think the return type here should be `Array`? Really minor but could we add this in? Thanks!

File: lib/services/cprsMetaService/gmvParameter.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                return callback(null, output);
+            })
+            .catch((err) => {
+                console.log(err);
+                callback(err);
+            });
+    }
+
+    // Argument 'ENTVAL' for entity value information
+    if (routineArg === 'ENTVAL') {
```
Comment (mfuroyama) [inline lib/services/cprsMetaService/gmvParameter.js]: Really minor, but consider changing this to an `else if`.

File: lib/services/cprsMetaService/gmvParameter.js [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+                });
+        } else if (entityArg === 'DIV') return { service: context.division.label, key: context.division.label };
+        else if (entityArg.match(/(\d+);?\D{3}?\((\d+)/)) {
+            const matched = entityArg.match(/(\d+);?\D{3}?\((\d+)/);
+            const fieldId = matched[1];
+            const fileNumber = matched[2];
+            const fileName = fileman.getFileLabelFromId(fileNumber).replace(/\s/g, '');
+
+            this.find(fileName, {
+                query: { id: `urn:vista:${context.stationId}:${fileNumber}:${fieldId}` },
```
Comment (mfuroyama) [inline lib/services/cprsMetaService/gmvParameter.js]: Not a big deal, and you don't have to, but you might want to consider using `serviceUtils.createURNFromParams`...totally up to you

File: lib/services/fileManService/getFieldFromFile.js [hunk 5]
```diff
@@ -2,11 +2,29 @@
 
 const vdmModels = require('../fileManService/vdmNonClinicalModel').vdmModel;
 const _ = require('lodash');
-
+// const util = require
```
Comment (mfuroyama) [inline lib/services/fileManService/getFieldFromFile.js]: Ahh, you can probably delete this line out

File: lib/broker/models/gmv-parameter.js (hunk 1 above)
Comment (sidjtd) [reply lib/broker/models/gmv-parameter.js]: DOH!

File: lib/services/cprsMetaService/gmvParameter.js (hunk 2 above)
Comment (sidjtd) [reply lib/services/cprsMetaService/gmvParameter.js]: Noooo these metadata entriessssss *cries*

**Pull request merged**